content/doc/tutorials/build-a-node-js-and-react-app-with-npm.adoc

```diff
@@ -277,28 +213,31 @@ your Pipeline, especially if it gains more complexity.
 . Save your edited `Jenkinsfile` and commit it to your local
   `simple-node-js-react-npm-app` Git repository. E.g. Within the
   `simple-node-js-react-npm-app` directory, run the commands: +
-  `git stage .` +
+  `git add .` +
   then +
-  `git commit -m "Add 'Test' stage"`
-. Go back to Jenkins again, log in again if necessary and ensure you've accessed
-  Jenkins's Blue Ocean interface.
-. Click *Run* at the top left, then quickly click the *OPEN* link which appears
-  briefly at the lower-right to see Jenkins running your amended Pipeline
-  project. If you weren't able to click the **OPEN** link, click the _top_ row
-  on the Blue Ocean interface to access this feature. +
-  *Note:* You'll notice from this run that Jenkins no longer needs to download
-  the Node Docker image. Instead, Jenkins only needs to run a new container from
-  the Node image downloaded previously. Also, from now on, no (new) `npm`
-  dependencies should need to be downloaded during the "Build" stage. Therefore,
-  running your Pipeline this subsequent time should be much faster. +
-  If your amended Pipeline ran successfully, here's what the Blue Ocean
-  interface should look like. Notice the additional "Test" stage. You can click
-  on the previous "Build" stage circle to access the output from that stage.
+  `git commit -m "Add 'Test' stage"` +
+and finally +
+`git push` to push your changes to your forked repository on GitHub, so it can be picked up by Jenkins.
+
+. In Jenkins, go back to *Dashboard* if necessary, then *simple-node-js-react-npm-app* and launch another build thanks to *Build Now*.
+. After a while, a new column *Test* appear in the *Stage View*.
+
 [.boxshadow]
-image:tutorials/node-js-react-11-test-stage-runs-successfully-with-output.png[alt="Test
-stage runs successfully (with output)",width=100%]
-. Click the *X* at the top-right to return to the main Blue Ocean interface.
+image:tutorials/node/test-stage.png[alt="Test stage",width=100%]
 
+. You can click on *#2* or on the number representing your last build on the left, under *Build History*. You will then see the details of the build.
++
+If Docker has not restarted since you last ran the Pipeline <<create-your-initial-pipeline-as-a-jenkinsfile,above>>, then no `npm` artifacts are downloaded during the "Build" stage.
+Therefore, running your Pipeline this subsequent time should be much faster.
+
+You can now click on *Pipeline Overview* to see the stages of the Pipeline.
```

Comment:
```suggestion
You can now select *Pipeline Overview* to see the stages of the Pipeline.
```
Suggestion to use ["select" instead of "click"](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click).